sgkit/variables.py

```diff
@@ -451,6 +451,15 @@ def _check_field(
     )
 )
 
+sample_cohort, sample_cohort_spec = SgkitVariables.register_variable(
+    ArrayLikeSpec(
+        "sample_cohort",
+        ndim=1,
+        kind="i",
+        __doc__="""The index of the cohort that each sample belongs to.""",
```

Comment:
Minor suggestion: mention that a negative value indicates a sample is not a member of any cohort.

Reply:
Good point - thanks!